src/main/java/core/basesyntax/Factorial.java

```diff
@@ -4,7 +4,8 @@
  * <p>Реализуйте метод getFactorial(int number) который вовзращает факториал числа.</p>
  */
 public class Factorial {
-    public int getFactorial(int number) {
-        return 0;
+    public int getFactorial(int number)
```

Comment:
```suggestion
    public int getFactorial(int number) {
```
curly bracket should be at the same line as a method name

Reply:
+

Reply:
agree